packages/shell-api/src/integration.spec.ts

```diff
@@ -1061,7 +1104,8 @@ describe('Shell API (integration)', function() {
     });
 
     describe('aggregate', () => {
-      describe('up to server 4.4', () => {
+      describe('server from 4.2.2 till 4.4', () => {
+        skipIfServerVersion(testServer, '< 4.2.2');
```

Comment:
We still want to test this on 4.0.x, I think.

```suggestion
      describe('server from 4.2.2 till 4.4 and 4.0.14 to 4.1', () => {
        skipIfServerVersion(testServer, '>= 4.2.0 < 4.2.2 || < 4.0.14');
```

Reply:
Yes, wanted to add it and forgot after the weekend. Thanks a lot for pointing at it!